dockerfiles/itest/itest/run_itest.sh

```diff
@@ -1,10 +1,20 @@
 #!/bin/bash
+set -vx
+set -e
+
+echo "installing paasta-tools (dependency of nerve-tools.)"
+PAASTA_VERSION=0.144.7
```

Comment:
TODO: bump this version to something that includes https://github.com/Yelp/paasta/pull/3450